### ISSUE TITLE
[BUG] non-intuitive behavior in tools/search_util function

### ISSUE THREAD

**Issue by Greenford** [GoogleCloudPlatform/dfcx-scrapi#24]
## Current Behavior

```
search = search_util.SearchUtil(creds_path=creds)
handlers = search.find_event_handlers()
```
2nd line throws `AttributeError: 'SearchUtil' object has no attribute 'agent_id'`
agent_id isn't a parameter in find_event_handlers. 

## Possible Solution

1. agent_id should be a constructor required param -or- 2. should be a parameter for find_event_handlers(). Recommend 2.

**Comment by kmaphoenix** [comment]
@Greenford Ack.

As mentioned in Issue #25 , the `SearchUtil` class could use a good overhaul.  
It appears the original author of these methods meant for the `agent_id` to be inherited via `self.agent_id`, but I agree that's non-intuitive.

`agent_id` should be passed as an arg to `find_event_handlers` similar to how `find_true_routes` and `find_list_parameters` work.

Link to method:
[find_event_handlers](https://github.com/GoogleCloudPlatform/dfcx-scrapi/blob/main/src/dfcx_scrapi/tools/search_util.py#L494)

**Comment by Greenford** [comment]
Fixed on [cleanup/search_util](https://github.com/GoogleCloudPlatform/dfcx-scrapi/tree/cleanup/search_util). File just needs linting and approval to merge.

**Comment by kmaphoenix** [comment]
Closing since this was fixed in PR #43 by @Greenford
